packages/webpack/template-webpack-plugin/test/cases/inline-scripts/inline-partial/index.js

```diff
@@ -0,0 +1,29 @@
+/*
+// Copyright 2024 The Lynx Authors. All rights reserved.
+// Licensed under the Apache License Version 2.0 that can be found in the
+// LICENSE file in the root directory of this source tree.
+*/
+/// <reference types="vitest/globals" />
+
+import { existsSync } from 'node:fs';
+import { readFile } from 'node:fs/promises';
+import { resolve } from 'node:path';
+
+import { foo } from './foo.js';
+import { bar } from './bar.js';
+
+it('should generate correct foo template', async () => {
```

Comment:
This test does fail without the change. Since the `app-service.js` is not actually run.

Reply:
I have added a `eval(manifest['/app-service.js'])` check to ensure the script is executed. This test now will fail on branch main (82405162) but success on current commit (84bac708)